src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -59,34 +61,17 @@
  * @since 1.8.0
  */
 public class ArgumentCaptor<T> {
-    
+
     HandyReturnValues handyReturnValues = new HandyReturnValues();
 
-    private final CapturingMatcher<T> capturingMatcher = new CapturingMatcher<T>();
+    private List<Object> arguments;
```

Comment:
I used Object in case the user will add two matcher that will capture incompatible elements. What do you think?